cmd/kubeadm/app/cmd/config_test.go

```diff
@@ -41,7 +44,7 @@ import (
 )
 
 const (
-	defaultNumberOfImages = 8
+	defaultNumberOfImages = 7
```

Comment:
what is the need for this change - is one of the images an empty line?

as a side note our unit tests should check for the proper existence of things like `pause`, `etcd` in the resulted list.
currently we don't have that.

Reply:
> what is the need for this change - is one of the images an empty line?

yes, it was. this change fixes that.

> as a side note our unit tests should check for the proper existence of things like pause, etcd in the resulted list.

makes sense. I can do it in a separate PR.